### ISSUE TITLE
Can't add multiple user-assigned managed id with new UI of Logic Apps Standard designer

### ISSUE THREAD

**Issue by ShingoHanai** [Azure/LogicAppsUX#2419]
### Describe the Bug with repro steps

1. Add multiple user-assigned managed id in [Settings - Identity] blade.
![image](https://github.com/Azure/LogicAppsUX/assets/66820701/0defa852-c5b4-4186-a83b-ed56d502894b)
2. Move to any workflow
3. Move to [Developer - Designer] blade
4. You can see error message "Invalid Number Of User Identities Multiple user identities are not supported. Please select only one user identity."
![image](https://github.com/Azure/LogicAppsUX/assets/66820701/90e94d88-158d-4451-bdd9-53292ca109b2)

### What type of Logic App Is this happening in?

Standard (Portal)

### Are you using Preview Designer or GA Designer

Preview

### Browser

Google Chrome

**Comment by hartra344** [comment]
@rllyy97 can you take a look?

**Comment by rllyy97** [comment]
@hartra344 This was implemented due to some errors we were getting from backend both in preview and in GA, logged in this GH issue: https://github.com/Azure/LogicAppsUX/issues/2076
It looks like these errors are not occurring in GA anymore, I get clarification from backend and remove the catch in Portal

**Comment by preetriti1** [comment]
If we do support multiple user assigned identities then we have to update all our designer code to accept multiple identities because that never does today and we didnt have code in old designer to handle that as well..
@ShingoHanai - was this working for you for connection creation earlier?

**Comment by ShingoHanai** [comment]
@preetriti1 
I can choose one user-assigned managed id in multiple user-assigned managed id list at HTTP action with old UI.

![image](https://github.com/Azure/LogicAppsUX/assets/66820701/78ee16f7-30e7-4d96-8344-c2f5ad4fa9c4)

**Comment by ShingoHanai** [comment]
I tested whether it works correctly from making HTTP action run with alternatively 2 user-assigned managed id and decoding received both access token.
As a result of this test, I found it works correctly and available to switch 2 managed id.
(object id is used each user-assigned managed id's.)

However performance it too slow.
![image](https://github.com/Azure/LogicAppsUX/assets/66820701/689b2bf4-edc5-4ff8-b9b1-0e1c9ed03d6b)
